README.md

```diff
@@ -80,6 +81,23 @@ All released versions can be found [here](https://github.com/sonatype-nexus-comm
 There are a range of options for installing the R plugin. You'll need to build it first, and
 then install the plugin with the options shown below:
 
+### Easiest Install
+
+Thanks to some upstream work in Nexus Repository (versions newer than 3.15), it's become a LOT easier to install a plugin. To install the `R` plugin, you can either build locally or download from The Central Repository:
+
+#### Option 1: Build locally
+* Clone this repo and `cd` to the cloned directory location
+* Build the plugin with `mvn clean package -PbuildKar`
+
+#### Option 2: Download from The Central Repository 
+* Download `nexus-repository-r-1.0.4-bundle.kar` from [The Central Repository](https://search.maven.org/artifact/org.sonatype.nexus.plugins/nexus-repository-r/1.0.4/bundle)
+* Copy the downloaded `nexus-repository-r-1.0.4-bundle.kar` file into the `<nexus_dir>/deploy` folder for your Nexus Repository installation.
```

Comment:
This is really a step for building locally, and for downloading, you might break it apart.

Reply:
You'll need to change the verbiage a bit to do so, FYI